docs/src/pages/components/masonry/BasicMasonry.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Masonry from '@material-ui/lab/Masonry';
+import MasonryItem from '@material-ui/lab/MasonryItem';
+
+export default function BasicMasonry() {
+  return (
+    <Box sx={{ width: 500, minHeight: 393 }}>
+      <Masonry columns={4} spacing={1}>
+        {heights.map((height, index) => (
+          <MasonryItem key={index}>
+            <Box
+              sx={{
+                textAlign: 'center',
+                height,
+                border: 1,
+                bgcolor: 'background.paper',
+              }}
+            >
+              {index + 1}
+            </Box>
+          </MasonryItem>
+        ))}
+      </Masonry>
+    </Box>
+  );
+}
+
+const heights = [150, 30, 90, 70, 110, 150, 130, 80, 50, 90, 100, 150, 30, 50, 80];
```

Comment:
cleaner.
```suggestion
import * as React from 'react';
import Box from '@material-ui/core/Box';
import Masonry from '@material-ui/lab/Masonry';
import MasonryItem from '@material-ui/lab/MasonryItem';

const heights = [150, 30, 90, 70, 110, 150, 130, 80, 50, 90, 100, 150, 30, 50, 80];

export default function BasicMasonry() {
  return (
    <Box sx={{ width: 500, minHeight: 393 }}>
      <Masonry columns={4} spacing={1}>
        {heights.map((height, index) => (
          <MasonryItem key={index}>
            <Box
              sx={{
                textAlign: 'center',
                height,
                border: 1,
                bgcolor: 'background.paper',
              }}
            >
              {index + 1}
            </Box>
          </MasonryItem>
        ))}
      </Masonry>
    </Box>
  );
}

```

Reply:
x the rest. cc @mnajdova

Reply:
Thanks! For the demos where the constant takes up 1 line, I did as you suggested, but for the others that take up many lines, I kept them as they were!